mvlearn/__init__.py

```diff
@@ -7,6 +7,8 @@
 import mvlearn.decomposition
 import mvlearn.preprocessing
 import mvlearn.merge
+import mvlearn.split
```

Comment:
I would prefer a compose module like sklearn https://github.com/scikit-learn/scikit-learn/tree/master/sklearn/compose
for what you have in merge and split.

Reply:
^ reduces the number of modules, since they aren't very large. I can update the references and changelog once this is made.

Reply:
I have made this change, and updated the changelog/references